tests/test_driven_controls.py

```diff
@@ -131,9 +126,6 @@ def test_driven_controls():
         _ = DrivenControl(rabi_rates=[-1])
```

Comment:
Not for this PR (and I know it wasn't you that added this test), but note that this approach to checking multiple calls for errors doesn't work -- the test will pass if *any* of the lines throw an exception.

Reply:
yes, this wrong pattern has been used a lot in the tests. We need to fix them in a seperate PR later.